conf/distro/tb-xen-distro.conf

```diff
@@ -0,0 +1,12 @@
+require conf/distro/include/tb-distro-common.conf
+
+DISTRO = "tb-xen-distro"
+DISTRO_FEATURES += "virtualization xen"
+
+IMAGE_INSTALL += " xen xen-tools"
+
+IMAGE_EFI_BOOT_FILES += " \
+    xen-${MACHINE};xen \
+"
+
+PREFERRED_VERSION_xen = "%+git"
```

Comment:
Following warning appears during build, suggesting it would not work:

```
WARNING: preferred version %+git of xen not available (for item xen)
WARNING: versions of xen available: 0.4.1+git 0.4.1+stable
```

Reply:
In the build logs, I can see that we are building this version of xen: `xen-tools-4.18+stable-r0`

This is not the one we wanted to use, right?

Reply:
Further investigation:

1. Xen itself (`xen` recipe) has been built from correct revision indeed: `aem_intel_fixes_v0.4.1`. The actual version selected is: `0.4.1+stable`
2. We build `xen-tools` recipe from upstream repo. We should rather build both of these from the same repo to ensure compatibility.

Reply:
Adjusted versions here: https://github.com/3mdeb/meta-trenchboot/pull/41/commits/29e1604dda6831f5c694a9acc3b82f587c2fad3d

It seems the `%` wildcard does not work as intended when used at the start.

The xen-tools are still built from upstream, not from our fork at this point, but this should not be a huge problem for TB demonstration at least.

Reply:
There are some tests that check the output of `xl dmesg`, although I'm not sure if this is part of xen-tools or something else.

Reply:
It is, but we still do include them (and `xl dmesg` works fine), we just do not build the tools from our fork right now, but from the "matching" (more or less) upstream release version.